GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -3064,15 +3064,22 @@ private void createWorktreeToolStripMenuItem_Click(object sender, EventArgs e)
         {
             using FormCreateWorktree formCreateWorktree = new(UICommands);
             DialogResult dialogResult = formCreateWorktree.ShowDialog(this);
-            if (dialogResult != DialogResult.OK || !formCreateWorktree.OpenWorktree)
+            if (dialogResult != DialogResult.OK)
             {
                 return;
             }
 
-            GitModule newModule = new(formCreateWorktree.WorktreeDirectory);
-            if (newModule.IsValidGitWorkingDir())
+            if (formCreateWorktree.OpenWorktree)
             {
-                SetGitModule(this, new GitModuleEventArgs(newModule));
+                GitModule newModule = new(formCreateWorktree.WorktreeDirectory);
```

Comment:
If this fails, nothing is opened, should be fine to resolve

Reply:
> If this fails, nothing is opened, should be fine to resolve

Do you mean the unchanged code should be wrapped in a `try {} catch{ RefreshRevisions(); throw; }`, too?

Reply:
No, that there is a risk for failure here, but we can live with that